### PR TITLE
Release 1.28.1

### PR DESCRIPTION
**Pull request by askur87**

### CODE REVIEW

File: build-system/publish-variable.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ext {
     groupIdSdk = 'com.midtrans'
-    versionSdk = '1.28.0'
+    versionSdk = '1.28.1-RC01'
```
Comment (uziwuzzy) [inline build-system/publish-variable.gradle:3]: still use RC?

Comment (askur87) [reply build-system/publish-variable.gradle:3]: ah... ok wi change it

**Review by theanh0512 — APPROVED** [review]
LGTM

File: build-system/publish-variable.gradle (hunk 1 above)
Comment (askur87) [reply build-system/publish-variable.gradle]: updated please check @uziwuzzy

**Review by uziwuzzy — APPROVED** [review]

**Pull request merged**